modules/efk-logging-deploy-subscription.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-deploy.adoc
+
+[id='efk-logging-deploy-subscription_{context}']
+= Installing the cluster logging subscription
+
+You can use the {product-title} console to subcribe to a cluster logging subscription, which creates the Cluster Logging Operator.
+
+.Procedure
+
+To subscribe to the subscription:
```

Comment:
We'll need to revisit this once various changes to OLM merge.  The workflow is different.